gulpfile.js

```diff
@@ -29,6 +29,16 @@ const paths = {
     ...IGNORED_PACKAGES,
   ],
   packageOther: ['packages/*/*/src/**/dev-prelude.js'],
+  packageFlow: [
+    'packages/core/cache/src/**/*.js',
+    'packages/core/core/src/**/*.js',
+    'packages/core/diagnostic/src/diagnostic.js',
+    'packages/core/fs/src/**/*.js',
+    'packages/core/package-manager/src/**/*.js',
+    'packages/core/plugin/src/PluginAPI.js',
+    'packages/core/utils/src/**/*.js',
+    'packages/core/workers/src/**/*.js',
+  ],
```

Comment:
So this basically means the package size will double since we will be publishing both src and compiled code?

Reply:
I'm afraid so...

Reply:
Too bad there's no way to bundle up flow definitions like we do for TS...